app/controllers/medicaid/income_introduction_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Medicaid
+  class IncomeIntroductionController < MedicaidStepsController
+    def step_class
+      NullStep
+    end
+
+    def update
```

Comment:
This looks like it inherits a ton of functionality from MedicaidStepsController just so that it can get the `next_path` method, and then overrides the rest of it. Makes me a little sad.

Reply:
also uses `edit` and `current_application` from there 😄

Reply:
yeah, but all of edit is unused, right? Would it break if you had a `def edit; end` there? It's a static page with no logic except for next_path.

Reply:
the main benefit we are getting from inheritance here is `current_application`. This is the way we are doing all of the "breather" pages so far -- do you think that re-doing the way do structure breather pages should block the merging of this PR?